test/LibraryManager.Test/Providers/Cdnjs/CdnjsProviderTest.cs

```diff
@@ -150,9 +150,7 @@ public void Cleanup()
 
             // Install library
             ILibraryOperationResult result = await _provider.InstallAsync(desiredState, CancellationToken.None).ConfigureAwait(false);
-            Assert.IsFalse(result.Success);
-            Assert.AreEqual(1, result.Errors.Count);
-            Assert.AreEqual("LIB007", result.Errors.First().Code);
+            Assert.IsTrue(result.Success);
```

Comment:
We calling InstallAsync from the provider itself, we know which provider to use.